exporter/collector/config_test.go

```diff
@@ -15,12 +15,19 @@
 package collector
 
 import (
+	"fmt"
 	"testing"
 
+	"go.opentelemetry.io/collector/component"
 	"go.opentelemetry.io/collector/confmap"
 	"gopkg.in/yaml.v3"
 )
 
+var testBuildInfo = component.BuildInfo{
```

Comment:
nit: consider moving this into the test function since it is only used there.  It makes it easier to read inputs and outputs when they are closer in the file.

Reply:
It was originally used in the other files (kind of hard to tell) but since I changed things to `exporter.Settings` that's not the case anymore so I'll fix this